python/ray/tests/test_actor.py

```diff
@@ -15,7 +15,260 @@
 import ray.cluster_utils
 
 
-def test_actor_exit_from_task(ray_start_regular):
+def test_caching_actors(shutdown_only):
```

Comment:
I'm guessing these are all moved to the beginning to avoid conflicting with the shared session?

Reply:
yep. I can add some comments on touched files if that would be helpful.